package.json

```diff
@@ -17,7 +17,7 @@
     "dist"
   ],
   "scripts": {
-    "demo-bundle": "yarn compile:cjs && browserify docs/js/index.js -o docs/demo-bundle.js",
+    "dev": "yarn compile:cjs -- --watch --environment SERVE,RELOAD,IS_CYPRESS_ENV:''",
```

Comment:
I'd prefer to keep `yarn start` as the defacto "run this thing so I can debug it" command.

How about we do this:

1. Make what is `yarn dev` now be `yarn start`
2. Make what is `yarn start` now (in this PR) be `yarn start:cypress`
3. Replace the instances of "start-server-and-test start" in this file to "start-server-and-test start:cypress"

WDYT?

Reply:
👍